src/Tizen.WebView/Tizen.WebView/PolicyDecisionEventArgs.cs

```diff
@@ -0,0 +1,189 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+
+namespace Tizen.WebView
+{
+    /// <summary>
+    /// Enumeration values used to specify Navigation Type.
+    /// </summary>
+    public enum NavigationType {
+        /// <summary>
+        /// Link clicked.
+        /// </summary>
+        LinkClicked = 0,
+        /// <summary>
+        /// Form submitted.
+        /// </summary>
+        FormSubmitted = 1,
+        /// <summary>
+        /// Back forward.
+        /// </summary>
+        BackForward = 2,
+        /// <summary>
+        /// Reload.
+        /// </summary>
+        Reload = 3,
+        /// <summary>
+        /// Form resubmitted.
+        /// </summary>
+        FormResubmitted = 4,
+        /// <summary>
+        /// Other.
+        /// </summary>
+        Other = 5
+    };
+
+    /// <summary>
+    /// Enumeration values used to specify Navigation Policy.
+    /// </summary>
+    public enum NavigationPolicy {
+        /// <summary>
+        /// Use.
+        /// </summary>
+        Use,
+        /// <summary>
+        /// Download.
+        /// </summary>
+        Download,
+        /// <summary>
+        /// Ignore.
+        /// </summary>
+        Ignore
+    };
+
+    /// <summary>
+    /// Arguments from the policy decision smart event callback.
+    /// This class also rovides the properties for Policy Decision of WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class PolicyDecisionEventArgs : EventArgs
```

Comment:
This class is used in various policy decision events.
I can't understand why do you provide `NavigationPolicy DecisionType` property?
It only valid for `NavigationPolicyDecideRequested`. 
What means of `DecisionType` in others event?

Reply:
So, are you suggesting that we should provide separated EventArgs for each event?

Reply:
Yes, you should separate a event args class